### PR TITLE
#164630360 Fix bug for distributing notification

### PR DESCRIPTION
**Pull request by danprocoder**
#### What does this PR do?
Fix bug for distributing notification

#### Description of Task to be completed?
Fix bug with distribution of notifications to users when people they follow posts a new article

#### How should this be manually tested?
1. Clone this branch.
2. Run `npm install` to install the dependencies.
3. Run `npm test` to run the tests.

#### What are the relevant pivotal tracker stories?
[#164630360](https://pivotaltracker.com/story/show/164630360)

### CODE REVIEW

File: server/controllers/article.js [hunk 1]
```diff
@@ -116,35 +107,60 @@ class ArticleController {
...
-          // Send email notification
-          HelperUtils.sendMail(follower.email,
-            'Authors Haven <no-reply@authorshaven.com>',
-            notification.message,
-            notification.message,
-            articleNotificationMarkup(`${author.firstname} ${author.lastname}`, article.title, article.description, notification.url));
-        }
-      });
+    const dbRows = []; // Database rows for users that opted for in-app notifications.
+    const emails = ['danielaustin718@gmail.com']; // Email addresses of users that want email notifications.
```
Comment (andela-gike) [inline server/controllers/article.js]: Why do you need to hard code your email address; kindly confirm

**Review by andela-gike — CHANGES_REQUESTED** [review]

File: server/controllers/article.js (hunk 1 above)
Comment (danprocoder) [reply server/controllers/article.js]: I added it so I could see what the email would look like. I forgot to remove it.

**Review by shaolinmkz — COMMENTED** [review]
REVIEWED

**Review by Dsalz — APPROVED** [review]
LGTM

**Review by andela-gike — APPROVED** [review]

**Pull request merged**